### ISSUE TITLE
New Admin User Keeps Getting Created at Every Restart/Start

### ISSUE THREAD

**Issue by war1000** [akaunting/docker#61]
Hi, I am using docker in unraid for akaunting. Every time I have to restart the container, it creates a new admin user using the default company email settings.

![image](https://user-images.githubusercontent.com/57807544/158070460-785ddda6-377a-43c8-9a85-cbe1a1ca8db2.png)

Here is the count as of today.

![image](https://user-images.githubusercontent.com/57807544/158070492-424ce6f6-dcba-4a1e-bd2d-fd0bb3d32b28.png)

If I set the following settings to false, it does not connect to the db I think anymore.

![image](https://user-images.githubusercontent.com/57807544/158070511-c457efdf-2d5f-452d-a8e6-4f9a57b7818b.png)

Would you kindly help?

Thanks!

**Comment by nightred** [comment]
I am having the same issue, and removing the variable has the system claim that there is no database.

```
[2022-04-09 03:47:33] production.ERROR: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'akaunting.ak_users' doesn't exist (SQL: select * from `ak_users` where `email` = office@castlecat.ca and `ak_users`.`deleted_at` is null limit 1) {"exception":"[object] (Illuminate\\Database\\QueryException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'akaunting.ak_users' doesn't exist (SQL: select * from `ak_users` where `email` = office@castlecat.ca and `ak_users`.`deleted_at` is null limit 1) at /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:712)
```

**Comment by azoller1** [comment]
I cant even get past the first initial login screen. I have the databes created but I get this error in the logs:

Base table or view not found: 1146 Table 'akounting.fhf_users' doesn't exist

**Comment by nightred** [comment]
Update on this issue.
Once I set the `DB_PREFIX` environment variable to an actual value, renamed all database tables with the prefix, I was able to set `AKAUNTING_SETUP` to false and still have everything work properly.

The documentation does not show this variable clearly, but it exists in the ENV example files.
Also the UNRAID template does not include this variable.

**Comment by nightred** [comment]
This only works on version 2, I cannot update to version 3 at this time so set your tag to use version 2 `akaunting/akaunting:2`
Stop your Akaunting docker image.
In your unraid template add the variable 'DB_PREFIX' and assign it a value like `mine_`
Set `AKAUNTING_SETUP` to `false`
Open a database tool (I used phpmyadmin) modify each table to have the prefix.
- Select the table, go to the Operations tab, and I used the move table option adding the prefix only.
Now restart the docker image and it should come back up.
